accel-pppd/ctrl/sstp/sstp.c

```diff
@@ -2705,6 +2686,8 @@ static void ssl_load_config(struct sstp_serv_t *serv, const char *servername)
 			goto legacy_ssl;
 	}
 
+	// cert is a reference. Do not free it.
+	cert = SSL_CTX_get0_certificate(ctx);
```

Comment:
we'll end here with no certificate loaded w/o ssl enabled (case with external tls termination) and break sstp handshake

Reply:
fixed